nodejs/week2/warmup/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const app = express();
+
+app.get("/" , (req, res) => {
+    res.send(`you are home!`);
+});
+
+app.get("/numbers/add/:id" , (req, res) => {
+    res.send(`you entere ${req.params.id}`);
+});
+
+//GET /numbers/multiply/<first number here>/<second number here>. in response send multiplication (first * second).
+app.use("/numbers/multiply" , (req, res) => {
+    const url = req.url;
+    const name = url.split("/");
+    res.send(name[name.length-2] + ' * ' + name[name.length-1]);
+});
+
+//GET /numbers/add?first=<number here>&second=<number here>. In response send sum (first + second).
+app.get("/numbers/add?" , (req, res) => {
+    res.send(`you entere ${req.query.firstNumber} ,  ${req.query.secoundNumber} and the summation is ${Number(req.query.firstNumber) + Number(req.query.secoundNumber)} `);
```

Comment:
If I follow the url as you described it in your comment as for example /numbers/add?first=5&second=2, the queries will not match with what you are sending back in the response. 
In res.send you are trying to send req.query.firstNumber and req.query.secoundNumber, however we only have these queries in the url: req.query.first and req.query.second. That is why we are getting an undefined. 
Otherwise it is good you converted the query strings into numbers with 'Number( )'